ipwb/util.py

```diff
@@ -144,6 +150,10 @@ def setIPWBReplayIndexPath(cdxj):
 
 def getIPWBReplayIndexPath():
     ipfsJSON = readIPFSConfig()
```

Comment:
The ipfs config needs to be read to check if the ipwb config option are present. If not, options are written and the variable refreshed with the new ipfs config containing the ipwb config options.